packages/core-data/src/entities.js

```diff
@@ -138,6 +138,7 @@ export const rootEntitiesConfig = [
 		name: 'user',
 		kind: 'root',
 		baseURL: '/wp/v2/users',
+		getTitle: ( record ) => record?.username || record?.slug,
```

Comment:
```suggestion
		getTitle: ( record ) => record?.name || record?.slug,
```

This matches the user data displayed in the editor by different components.